src/pages/Product/index.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react'
```

Comment:
you can import only the things you need

Reply:
Yes, i have used eslint config for that, React was needed in that case.